File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.cpp
class Solution {
public:
    int countPairs(vector<int>& nums, int target) {
        int res = 0;
        int n = nums.size();
        sort(nums.begin(), nums.end());
        for (int i = 0; i < n; i++) {
            for (int j = i + 1; j < n; j++) {
                if (nums[i] + nums[j] < target) res++;
                else break;
            }
        }
        return res;
    }
};

File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.py
class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        n = len(nums)
        nums.sort()
        res = 0
        for i in range(n):
            for j in range(i+1,n):
                if nums[i] + nums[j] < target:
                    res += 1
                else:
                    break
        return res